pnpm build fails

after `pnpm install` I end up with `./node_modules/.bin/tsc  --version` => `Version 5.0.4` which fails in `pnpm build`:

```
> gathio@1.3.0 build /srv/gathio
> tsc

src/activitypub.js(1,21): error TS7016: Could not find a declaration file for module 'request'. '/srv/gathio/node_modules/.pnpm/request@2.88.2/node_modules/request/index.js' implicitly has an 'any' type.
  If the 'request' package actually exposes this module, consider sending a pull request to amend 'https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/request'
src/activitypub.js(6,26): error TS7016: Could not find a declaration file for module 'sanitize-html'. '/srv/gathio/node_modules/.pnpm/sanitize-html@2.10.0/node_modules/sanitize-html/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/sanitize-html` if it exists or add a new declaration (.d.ts) file containing `declare module 'sanitize-html';`

```

what version of tsc is needed?

--- a/package.json
+++ b/package.json
@@ -45,6 +45,6 @@
     "eslint": "^8.40.0",
     "nodemon": "^2.0.22",
     "prettier": "^2.8.8",
-    "typescript": "^5.0.4"
+    "typescript": "5.1.6"
   }
 }

